src/ship_routing/core.py

```diff
@@ -1,4 +1,4 @@
-from dataclasses import dataclass
+from dataclasses import dataclass, asdict
```

Comment:
`asdict`  never used?
```suggestion
from dataclasses import dataclass
```